common/lib/capa/capa/templates/jsinput.html

```diff
@@ -1,3 +1,4 @@
+<%page expression_filter="h"/>
 <section id="inputtype_${id}" class="jsinput"
     data="${gradefn}"
     % if saved_state:
```

Comment:
Hmm, I'm not sure `|x` is right in an HTML file.  Do we know why it needs XML escaping?